src/nvidia-cuda/devcontainer-feature.json

```diff
@@ -10,14 +10,27 @@
       "default": false,
       "description": "Additionally install CUDA Deep Neural Network (cuDNN) shared library"
```

Comment:
Can we bump the minor version? Thanks! 

https://github.com/devcontainers/features/blob/50a7b160f0c7ea1956e7897f5843ef57332e6141/src/nvidia-cuda/devcontainer-feature.json#L3

Reply:
I used 1.1.0 as the new version. Is this correct? I should use 1.0.8?